GCRCatalogs/dc2_object.py

```diff
@@ -678,7 +655,10 @@ def _subclass_init(self, **kwargs):
         super()._subclass_init(**kwargs)
 
     def _detect_available_bands(self):
-        return [col.rpartition('_')[0] for col in self._columns if col.endswith('_FLUXMAG0')]
+        return (
+            [col.rpartition('_')[0] for col in self._columns if col.endswith('_FLUXMAG0')] or
+            [col.rpartition('_')[2] for col in self._columns if col.startswith('psFlux_')]
+        )
```

Comment:
I would suggest using a couple of lines to write out the logic here and maybe even insert a sanity check.  Clarify that the two different column names possible are from different versions and  then do something different based on the version.
In theory, this isn't robust against there being mixed columns, but in practice if that happens perhaps something should be flagged.